API.md

```diff
@@ -1,19 +1,349 @@
-# Party Service API
+###Party Service API
```

Comment:
not sure why this isn't considered the main heading/title of the document (i.e. a single hash)